### ISSUE TITLE
Npm not found

### ISSUE THREAD

**Issue by laserblue** [nodeschool/discussions#2461]
If you have a problem with an error message:
Aop: Dory d/node.js
* node version: v6.11.4
* npm version:  3.10.10
* os: Android 6.0

### Error output
``/system/bin/sh: how-to-npm: not found
 /```
To get the Nodeschool workshops running on Dory d/nodejs on Android 4, I had to set an environment variable but I have forgotten what it was.

### My Code
```js
 // copy your code here
```

**Comment by victorperin** [comment]
what is the output of:
`npm i -g how-to-npm`?

After running this command, does `how-to-npm` works?

**Comment by laserblue** [comment]
![Screenshot_2019-12-12-21-26-35](https://user-images.githubusercontent.com/7808357/70849259-46eb0d00-1e39-11ea-8e01-87244ff0da12.png)
App: Dory d/node.js

node version: v6.11.4
npm version: 3.10.10
os: Android 6.0

To keep you informed until I get the data into a form I can add here:
The output of npm i -g how-to-npm was 2 warnings (sprintfs deprecated, another file deprecated) and a long dependency tree. I don´t know how to pipe the output to a file by using > so I took a half dozen screenshots that I will add asap. Dory node.js has a copy feature that does not work properly to get shell command line output. There does not seem to be a cursor to select text so I usually get only the last 10 or 20 lines or so when I press copy.

I was surprised, however, not to see the usual permission denied warnings that usually occur with a global install on Dory d/node.js because I don have a rooted phone. 
I did see from the path output that my path is messed up. Thatś one thing I need to relearn how to change on Android.
Anyway, after the install, nothing changed. Entering how-to-npm just returns /system/bin/sh: how-to-npm: not found


Error output
``/system/bin/sh: how-to-npm: not found
